sys-apps/debianutils/debianutils-5.4.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+DESCRIPTION="A selection of tools from Debian"
+HOMEPAGE="https://packages.qa.debian.org/d/debianutils.html"
+SRC_URI="mirror://debian/pool/main/d/${PN}/${PN}_${PV}.orig.tar.xz"
+
+LICENSE="BSD GPL-2 SMAIL"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~m68k ~mips ~ppc ~ppc64 ~riscv ~s390 ~sparc ~x86 ~x86-linux"
+IUSE="+installkernel static"
+
+PDEPEND="
+	installkernel? (
```

Comment:
I just have a doubt on this dependency:
1. It's a `PDEPEND` one
2. I'm not sure we need `installkernel` script to install kernel, it's already handled by the kernel ebuild itself I guess ?

Reply:
> I just have a doubt on this dependency:

I had my doubt about why we even have these debianutils in SDK. Personally I'd just drop the package, but maybe someone is actually using something from it. [The debianutils package website](https://packages.debian.org/sid/debianutils) tells me that it contains the following tools: add-shell installkernel ischroot remove-shell run-parts savelog update-shells which. Of those, probably `which` is "interesting", but it could be easily replaced with bash's builtin `command -v`.

> 
>     1. It's a `PDEPEND` one

And what's wrong with it? There must be something I'm not getting.

>     2. I'm not sure we need `installkernel` script to install kernel, it's already handled by the kernel ebuild itself I guess ?

We always had an (most likely broken anyway) `installkernel` utility in the SDK, but with the update, it seems to be split out into a separate package. So I was basically trying to preserve what we had so far in SDK.

I guess I could either force disable the installkernel USE flag, or just drop this package altogether. What do you think?

Reply:
> installkernel utility in the SDK, but with the update, it seems to be split out into a separate package.

Oh ok, then it might be safe to keep it - we can try to remove it in a following PR to not block this one furthermore. :) 

> And what's wrong with it? There must be something I'm not getting.

To me, `PDEPEND` means "this dependency does not impact the build / the run of this program" so I doublecheck the utility of a dependency with it's defined as `PDEPEND`

Reply:
> > installkernel utility in the SDK, but with the update, it seems to be split out into a separate package.
> 
> Oh ok, then it might be safe to keep it - we can try to remove it in a following PR to not block this one furthermore. :)
> 

I'll try to drop the debianutils package later - I've seen a use of `which` in our build scripts, so maybe those will need a migration. In general I'd say that our SDK needs a check at what it's pulling, because it seems to be that we carry there some unnecessary stuff.

> > And what's wrong with it? There must be something I'm not getting.
> 
> To me, `PDEPEND` means "this dependency does not impact the build / the run of this program" so I doublecheck the utility of a dependency with it's defined as `PDEPEND`

Ah, thanks! Will keep that in mind.